discovery_engine_core/ai/bert/examples/bert.rs

```diff
@@ -12,21 +12,18 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-//! Run as `cargo run --example mbert
+//! Run as `cargo run --example bert
 
-use xayn_discovery_engine_bert::{FirstPooler, SMBert, SMBertConfig};
-use xayn_discovery_engine_test_utils::smbert;
+use xayn_discovery_engine_bert::{Config, FirstPooler};
+use xayn_discovery_engine_test_utils::asset::smbert;
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    let mbert = SMBertConfig::from_files(smbert::vocab()?, smbert::model()?)?
-        .with_pooling::<FirstPooler>()
+    let pipeline = Config::new(smbert()?)?
         .with_token_size(64)?
+        .with_pooler::<FirstPooler>()
         .build()?;
-    let embedding = mbert.run("This is a sequence.")?;
-    let size = SMBert::embedding_size();
-
+    let embedding = pipeline.run("This is a sequence.")?;
     println!("{}", *embedding);
-    assert_eq!(embedding.shape(), [size]);
```

Comment:
why we're removing this assert?

Reply:
seems that this got lost in between, i'll add it again, good catch 👍